### DISCUSSION TITLE
Are intel gpus supported?

### DISCUSSION THREAD

**Discussion by Nesuhst** [lllyasviel/Fooocus#1754] [Q&A]
Seems like they're getting more competitive nowadays in terms of price to vram. I'd recently seen some $220 deals for a a770 16gb vram card and am thinking of picking one up to upgrade my system, but I haven't seen anything regarding intel's gpus in the supported list. I've only seen nvidia and amd. So, if it's supported, that'll tip the balance in terms of what I might get next. Maybe either the a770 16gb or I might see how the battlemage series card shifts things in the market and try that type if it's a better option.

**Comment by mashb1t** [comment]
We do currently not offer support for Intel ARC GPUs, but they might work with DirectML.
References:
- https://www.reddit.com/r/IntelArc/comments/11an12q/stable_diffusion_web_ui_for_intel_arc/
- https://community.intel.com/t5/Blogs/Tech-Innovation/Artificial-Intelligence-AI/Intel-and-Microsoft-Collaborate-to-Optimize-DirectML-for-Intel/post/1542055
- https://devblogs.microsoft.com/directx/dml-extension-a1111webui/

Please take this info with a grain of salt and do your own research before buying the GPU!

**Reply by Nesuhst** [reply]
Darn. Thanks for the info and links. The gpus look like such a good deal too. Ah well. 

I've seen some youtube videos on the intel youtube channel about stable diffusion working on intel gpus some time ago but didn't remember the details clearly and I don't understand all this stuff perfectly.  

https://www.youtube.com/watch?v=5X0RmlH6JI4&t=41s

The intel gpu team seems like they might be flexible and responsive enough to quickly add updates. Maybe the fooocus team and the intel team can contact eachother and work together? I don't know about all this stuff. Just a thought.

**Comment by Nesuhst** [comment]
After searching around a bit, I found this "Intel arc graphics thread" from the comfyui community on the discussions tab over on their community. I'm not savvy enough, but it seems like they have it working. Maybe this could be of interest for the fooocus team to draw from for future intel arc gpu fooocus use potential.

https://github.com/comfyanonymous/ComfyUI/discussions/476

**Comment by Chimrod** [comment]
Hello, I confirm I can use Fooocus with my intel Arc 770. I’m using debian with IPEX, and it just work out of the box.
